tests/lib/snaps/test-snapd-mokutil/snapcraft.yaml

```diff
@@ -0,0 +1,23 @@
+name: test-snapd-mokutil
```

Comment:
I agree with @stolowski, if we aren't using test-snapd-mokutil here, we should leave this out until the test actually uses it

Reply:
I'll add the change in teh test to start using the snap, I already pushed the snap